data/sql_updates/rad_analyst.sql

```diff
@@ -16,6 +16,7 @@ select
     -- I am not sure we want to filter on this, if the name is different here it will behave differently than the other committee searches by name
     cmte_nm as committee_name,
     anlyst_id as analyst_id,
+    cast(anlyst_short_id as numeric) as analyst_short_id,
```

Comment:
Both IDs are numbers and the other ID is numeric typed, so I thought the display of numeric ids should be consistent.